lib/active_merchant/billing/gateways/kushki.rb

```diff
@@ -4,8 +4,8 @@ class KushkiGateway < Gateway
       self.display_name = 'Kushki'
       self.homepage_url = 'https://www.kushkipagos.com'
 
-      self.test_url = 'https://api-uat.kushkipagos.com/v1/'
-      self.live_url = 'https://api.kushkipagos.com/v1/'
+      self.test_url = 'https://api-uat.kushkipagos.com/'
+      self.live_url = 'https://api.kushkipagos.com/'
```

Comment:
Per Kushki (https://docs.kushkipagos.com/api/card/requestcardtoken), some API calls go to `card/v1` whereas some just go to `v1`. I refactored this while attempting to fix something that turned out to be unrelated, but left the changes to more closely adhere to Kushki's docs.

Reply:
That's only slightly annoying.